README.md

```diff
@@ -1 +1,14 @@
-A base repo for all microservices
+# A base repo for all microservices
+
+### Running locally using docker compose
+#### Option 1 - All services
+```sh
+export PROFILE=dev
+docker-compose up -d
+```
+
+#### Option 2 - All services in local mode, without service discovery nor config
+```sh
+export PROFILE=local
```

Comment:
It looks like this says that setting the env var to local changes how docker-compose brings up services. Is that true? I think that it you call up without names of services everything comes up.

Reply:
yes not accurate. all are up but not configured...
will rephrase